src/clipboard.re

```diff
@@ -0,0 +1,5 @@
+[@bs.module "react-native"] [@bs.scope "Clipboard"]
+external getString : unit => Js.Promise.t(string) = "";
```

Comment:
Does it ever throw?

Reply:
it doesn't, and it returns an empty string (in Promise) if there's nothing in the clipboard